### ISSUE TITLE
shouldn't this be await resolve(event) ?

### ISSUE THREAD

**Issue by jli001L** [huntabyte/sk-supabase-auth#3]
I was watching your videos on youtube and got two questions I couldn't figure by googling etc.

1. 

https://github.com/huntabyte/sk-supabase-auth/blob/03422641c7c66ed5ebc8c64a32cdc8d7c7f7f63a/src/hooks.server.ts#L11

Shouldn't this be 
```ts
return await resolve(event)
```
or doesn't it make any difference ?

2.

https://github.com/huntabyte/sk-supabase-auth/blob/03422641c7c66ed5ebc8c64a32cdc8d7c7f7f63a/src/hooks.client.ts#L1

What is this hook.client.ts file good for? Does it even do anything?

Thank you upfront 🙂 and thank you for your great videos and all your efforts. Keep it up 👍

**Comment by huntabyte** [comment]
I don't think this makes a difference unless I was assigning the return value to a variable like `response` which I was then going to take action on.

**Comment by jli001L** [comment]
Yes, that makes sense. Thank you for clarifying. 

Could you please also elaborate on: 

2. *What is this hook.client.ts file good for? Does it even do anything?*

Thank you
